app/javascript/asset-comments/components/AssetComments.spec.js

```diff
@@ -51,10 +51,10 @@ describe('AssetComments', () => {
     // checking sort of comments, should be re-ordered
     expect(wrapper.find('.comments-list').findAll('li').wrappers[0].text()).toContain('This is also a comment')
     expect(wrapper.find('.comments-list').findAll('li').wrappers[0].text()).toContain('Jane Smythe (js2)')
-    expect(wrapper.find('.comments-list').findAll('li').wrappers[0].text()).toContain('30 September 2017, 12:18')
+    expect(wrapper.find('.comments-list').findAll('li').wrappers[0].text()).toContain('30 September 2017 at 12:18')
     expect(wrapper.find('.comments-list').findAll('li').wrappers[1].text()).toContain('This is a comment')
     expect(wrapper.find('.comments-list').findAll('li').wrappers[1].text()).toContain('John Smith (js1)')
-    expect(wrapper.find('.comments-list').findAll('li').wrappers[1].text()).toContain('31 August 2017, 11:18')
+    expect(wrapper.find('.comments-list').findAll('li').wrappers[1].text()).toContain('31 August 2017 at 11:18')
```

Comment:
I have seen this date formatting issue before. What changes this? Will it change again in a later update?